db/schema.rb

```diff
@@ -753,6 +753,7 @@
     t.datetime "updated_at", precision: 6, null: false
     t.uuid "delivery_partner_id", null: false
     t.uuid "cohort_id"
+    t.integer "uploaded_urns", array: true
```

Comment:
Should this be a `string` array to match what a URN currently is/prevent issues with typos trailing zeroes ?

Reply:
Good call, will switch to string. I came at it purely from a GIAS point of view.